File: 2/PDOO/NapakalakiRuby/lib/cultist.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
module NapakalakiGame
class Cultist
  attr_reader :gainedLevels
  def initialize(name,gainedLevels)
    @name=name
    @gainedLevels=gainedLevels
  end
end
end

File: 2/PDOO/NapakalakiJava/src/napakalakiGame/Cultist.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalakiGame;

/**
 *
 * @author alejandro
 */
public class Cultist {
    private String name;
    private int gainedLevels;
    
    public Cultist(String name,int gainedLevels){
        this.name=name;
        this.gainedLevels=gainedLevels;
    }
    
    public int getGainedLevels(){
        return gainedLevels;
    }
}
